test/test-search.ts

```diff
@@ -0,0 +1,105 @@
+import test from 'ava';
+import assert from 'assert';
+import * as search from '../src/search';
+
+test(
+    'searchValue should have http module in standard require' +
+        'http case',
+    async t => {
+      const content = 'const a = require(\'http\');';
+      const result = await search.search(content);
+      t.true(result.requiredModules.has('http'));
+    });
+    
+test(
+    'searchValue should not have http module when there are no require calls',
+    async t => {
+      const content = 'console.log(\'http\')';
+      const result = await search.search(content);
+      t.true(result.requiredModules.size === 0);
```

Comment:
If the statement is just a test of equality, we should use `t.equals` instead.

Reply:
done